common/environment/setup/sourcepkg.sh

```diff
@@ -3,7 +3,7 @@
 
 ## VARIABLES
 unset -v pkgname version revision short_desc homepage license maintainer
-unset -v archs distfiles checksum build_style build_helper nocross broken
+unset -v archs distfiles checksum build_style build_helper nocross disable_parallel_check broken
```

Comment:
Please move to near `disable_parallel_build`

Reply:
Fixed